api/turing/config/config.go

```diff
@@ -332,6 +346,24 @@ type ClusterConfig struct {
 	InClusterConfig bool
 	// VaultConfig is required if InClusterConfig is false.
 	VaultConfig *VaultConfig `validate:"required_without=InClusterConfig"`
```

Comment:
Should be removed too

Reply:
Yep, will remove this together with unit tests changes, as this field is still referenced there.